Prose/ProseLib/Sources/ProseCoreTCA/Types/ChatState.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+import ProseCoreClientFFI
+
+public struct ChatState: Equatable {
+    public var kind: Kind
+    public var timestamp: Date
+
+    public init(kind: ChatState.Kind, timestamp: Date) {
+        self.kind = kind
+        self.timestamp = timestamp
+    }
+}
+
+public extension ChatState {
+    /// See: https://xmpp.org/extensions/xep-0085.html
```

Comment:
Just for your information: the Swift markup language supports `- SeeAlso:`, but Xcode doesn't show it in Quick Help…
I think DocC supports it, but as I've not been able to compile the Prose documentation, I'm not sure.